osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -425,19 +430,23 @@ namespace renderkit {
             m_displayWidth = widthOverride;
             m_displayHeight = heightOverride;
         }
+
+#ifdef OSVR_RM_USE_OPENGLES20
+        m_GLVAOExtensionAvailable = IsGLExtensionSupported("GL_OES_vertex_array_object");
+        m_GLDiscardExtensionAvailable = IsGLExtensionSupported("GL_EXT_discard_framebuffer");
+#endif
     }
 
     RenderManagerOpenGL::~RenderManagerOpenGL() {
         if (m_displayOpen) {
-            for (size_t i = 0; i < m_frameBuffers.size(); i++) {
+            for (size_t i = 0; i < GetNumDisplays(); i++) {
```

Comment:
Given that we're deleting the elements of the m_frameBuffers vector here, it seems to make more sense to iterate based on its actual size than based on something that may or may not be its size, even if GetNumDisplays() just calls m_frameBuffers.size() now because it will keep the code from drifting.  What's better about using the different method here?

Reply:
m_frameBuffers elements aren't deleted in this loop - it just attempts to make each display current via the toolkit and if any of them fail, it bails out early. 

That being said, the loop that does delete elements from m_frameBuffers re-uses a loop iterating over m_colorBuffers. I'll separate out those deletes into a separate loop.

Reply:
OK, done.